packages/manager/modules/account-migration/src/notification/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,34 @@
+{
+  "ovh_manager_account_migration_pending_contracts1": "Nous avons modifié nos Conditions d’utilisation. Vous avez jusqu’au {{date}} pour",
+  "ovh_manager_account_migration_accept_contracts": "accepter ces nouveaux contrats.",
+  "ovh_manager_account_migration_accept_all_contracts": "accepter tous vos contrats en attente.",
+  "ovh_manager_account_migration_pending_contracts2": "Passé cette date vous ne pourrez plus renouveler vos services. Pour plus d’informations sur les changements apportés,",
+  "ovh_manager_account_migration_faq_text": "consultez la FAQ",
+
+  "ovh_manager_account_migration_pending_contracts_success": "La mise à jour de votre compte a été enregistrée, vous recevrez un email à la fin de cette opération, merci de patienter. Pour plus d’informations sur les changements apportés, ",
+
+  "ovh_manager_account_migration_pending_debts1": "Cependant, vous avez actuellement une dette de {{debt}}. Nous vous invitons à la régler dans la rubrique ",
+  "ovh_manager_account_migration_pending_debts2": "pour pouvoir accepter les nouveaux contrats",
+
+  "ovh_manager_account_migration_pending_order": "Cependant, vous avez actuellement une commande en cours. Nous vous invitons à la régulariser dans la rubrique",
+
+  "ovh_manager_account_migration_pending_contact": "Cependant, vous avez actuellement un changement de contact en cours sur l'un de vos services. Nous vous invitons à finaliser l'opération dans la rubrique",
+
+  "ovh_manager_account_migration_pending_debts_and_order1": "Cependant, vous avez actuellement une dette de {{debt}} et une commande en cours. Nous vous invitons à régler ces opérations dans les rubriques ",
+  "ovh_manager_account_migration_pending_debts_and_order2": "pour pouvoir accepter les nouveaux contrats",
+
+  "ovh_manager_account_migration_pending_debts_and_contact1": "Cependant, vous avez actuellement une dette de {{debt}}. Nous vous invitons à régler ces opérations dans les rubriques",
+  "ovh_manager_account_migration_pending_debts_and_contact2": "pour pouvoir accepter les nouveaux contrats",
+
+  "ovh_manager_account_migration_pending_order_and_contact1": "Cependant, vous avez actuellement une commande et un changement de contact en cours. Nous vous invitons à régler ces opérations dans les rubriques ",
+  "ovh_manager_account_migration_pending_order_and_contact2": "pour pouvoir accepter les nouveaux contrats",
+
+  "ovh_manager_account_migration_pending_debts_order_and_contact1": "Cependant, vous avez actuellement une dette de {{debt}}, une commande et un changement de contact en cours.  Nous vous invitons à régler ces opérations dans les rubriques",
+  "ovh_manager_account_migration_pending_debts_order_and_contact2": "pour pouvoir accepter les nouveaux contrats. ",
+
+  "ovh_manager_account_migration_and": "et",
+
```

Comment:
This looks like a lot of chunks to translate instead of full sentences. It might be difficult for translation team to translate those correctly

Reply:
There are 3 conditions, which can occur in various different combinations (resulting in 9 combinations). Hence, we will have to generate many translations with the word `et` in them. Please suggest if you feel there is a good method to handle this

Reply:
I think we might have to fully write all translations for all cases, as it probably makes sense in french and even english but maybe not polish or spanish

Reply:
I have merged whatever is possible, into single sentences, except for links that require tracking too. Thanks for the suggestion :)